x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/PositionToXContent.java

```diff
@@ -89,7 +89,7 @@ protected XContentBuilder valueToXContent(XContentBuilder builder, ToXContent.Pa
                     return builder.value(unsignedLongAsNumber(l));
                 }
             };
-            case KEYWORD, TEXT -> new PositionToXContent(block) {
+            case KEYWORD, SEMANTIC_TEXT, TEXT -> new PositionToXContent(block) {
```

Comment:
Minor suggestion, swap `SEMANTIC_TEXT` and `TEXT`.

Reply:
see https://github.com/elastic/elasticsearch/pull/113920#discussion_r1806171017